lib/processor.js

```diff
@@ -29,9 +30,11 @@ function parse(text) {
         column = 0,
         mode = modes.NORMAL,
         next,
-        expErr;
+        expErr,
+        expErrQty;
 
-    var EXP_ERR_REGEX = /\/\*\s*error\s+(.+?)\s*\*\//g;
+    var EXP_ERR_REGEX = /\/\*\s*error\s+(.+?)\s*\*\//g,
+        EXP_ERR_QTY_REGEX = /\/\/\s*(.+?) errors\s*/;
```

Comment:
Right now the quantity regex would match `var str = "// 2 errors";`. Adding an end of string assertion (`$`) will make that harder to hit.